spec/models/slice/as_json_spec.rb

```diff
@@ -4,8 +4,7 @@
   describe "#as_json" do
 
     let :slice do
-      Slice.new(title: 'Title',
```

Comment:
It would be useful to keep a spec to ensure that attributes defined in [slices](https://github.com/withassociates/slices/blob/master/spec/dummy/app/slices/title/title_slice.rb#L2) are included when the slice is serialized.